sphinx/environment/__init__.py

```diff
@@ -562,9 +561,8 @@ def read_doc(self, docname, app=None):
         """Parse a file and add/update inventory entries for the doctree."""
         self.prepare_settings(docname)
 
+        # Add srccdir/docutils.conf to dependencies list if exists
         docutilsconf = path.join(self.srcdir, 'docutils.conf')
-        # read docutils.conf from source dir, not from current dir
-        OptionParser.standard_config_files[1] = docutilsconf
```

Comment:
Sphinx only overrides `standard_config_files[1]`.
The original value is here:
```
    standard_config_files = [
        '/etc/docutils.conf',           # system-wide
        './docutils.conf',              # project-specific
        '~/.docutils']                  # user-specific
```

Should we support both system-wide and user-specific docutils.conf?

Reply:
I think this would be a good idea, but it might be confusing for some users as it will change behaviour. Maybe add a TODO to do this for 2.0, which is a major change and will allow us to do such a thing.

Reply:
My last comment is not clear. What I want to say is " is this support really needed?".
I think system-wide and user-specific configuration changes output only locally. It means the outputs can't be reproduced on other environments.

In addition, our document does not mention about both of them. It only mentions about project-specific one.

>An optional file docutils.conf can be added to the configuration directory to adjust Docutils configuration if not otherwise overriden or set by Sphinx.
http://www.sphinx-doc.org/en/master/config.html#module-conf

Reply:
Oh, then my comment still stands but in reverse. I think we _should_ use them because that's what docutils does, but if we remove this feature then we should wait for a major version bump.

Reply:
For document builds, I think that the same product should be created independent of the environment. However, if `/etc/docutils.conf` is referenced, different results may be generated for each environment. It is good to refer only to the files included in the Sphinx project.

Reply:
I noticed `OptionParser` also refers `$DOCUTILSCONFIG` envvar. docutils gives priority to it rather than `standard_config_files`. It means `{confdir}/docutils.conf` might be ignored if `$DOCUTILSCONFIG` is set.

I think it is not good for Sphinx. We should find docutils.conf with our own searching rules.